setup.cfg

```diff
@@ -1,2 +1,7 @@
 [metadata]
 description-file = README.md
+
+[flake8]
+max-line-length = 119
+ignore = F401,E731
```

Comment:
### Suggestion:
Maybe add rules description (inline or in one block, whatever), like:
```
# F401 - Module imported but unused
# E731 - Do not assign a lambda expression, use a def
# Source: https://lintlyci.github.io/Flake8Rules/
```

Reply:
fixed in the following commit